ntfysh_client/SettingsDialog.cs

```diff
@@ -37,5 +73,20 @@ private void cancelButton_Click(object sender, EventArgs e)
         {
             DialogResult = DialogResult.Cancel;
         }
+
+        private void setNotificationsUIElements()
```

Comment:
Nitpick: Naming convention is capitalised first letter

Reply:
renamed